doc/nrf/ug_matter_overview_network_topologies.rst

```diff
@@ -0,0 +1,147 @@
+After exchanging the keys and establishing secure channel, the commonly available AES modes of operation are used to provide shared key cryptographic operations.
+
+.. _ug_matter_network_topologies_commissioning:
+
+Matter network commissioning
+****************************
+
+The commissioning in Matter is a process of joining a device to a fabric (operational network).
+The process takes place between a commissioner device and a commissionee device.
+It consists of the following stages:
+
+1. Device discovery -- The commissioner discovers devices that can be commissioned onto the network.
+   If the node is being added to its first Matter fabric, the device discovery uses Bluetooth LE.
+#. Security setup with PASE -- The commissioner runs the Passcode-Authenticated Session Establishment (PASE) protocol, which is exclusive to the commissioning process.
+   This protocol is used to establish the first session between devices that take part in commissioning.
+   The session is established with a passcode provided out-of-band and that is used to derive encryption keys.
+   This passcode is known only to the commissioner and the commissionee.
+#. Establishing fail-safe -- The commissioner requests the commissionee to back up its original configuration.
+#. Preliminary node configuration -- The commissioner configures the commissionee with regulatory information and the current UTC time.
+#. Device Attestation Certificate verification -- The commissioner checks whether the commissionee is a certified Matter device.
+   If this verification fails, the commissioner can either terminate or continue the commissioning procedure.
+   The verification can fail if the device is not able to prove the validity and ownership of mandatory :ref:`ug_matter_device_attestation` elements.
+#. Installing operational credentials -- The commissioner installs Node Operational Certificate (NOC) and Operational ID on the commissionee.
+   The commissionee becomes the new node of the Matter fabric.
+#. Network commissioning -- The commissioner provisions the commissionee node with the operational network credentials, either Wi-Fi or Thread, and requests the commissionee to connect to the network.
+#. Operational discovery -- The commissioner discovers the commissionee node on the operational network using DNS-SD.
+   This way, the commissioner learns the IP address of the node.
+#. Security setup with CASE -- The commissioner and the node use the Certificate-Authenticated Session Establishment (CASE) protocol to establish secure communication.
+   The CASE protocol is in charge of exchanging NOCs to set up a session secured with a new pair of keys.
+   The CASE connection is reset each time a device breaks the connection.
+#. Disarming fail-safe -- The commissioner requests the commissionee node to remove the stored configuration backup.
+#. Message exchange -- The commissioner and the commissionee exchange a CASE-encrypted message on the operational network.
```

Comment:
```suggestion
#. Message exchange -- The commissioner and the commissionee exchange a AES-encrypted message on the operational network.
```
This sounded as if CASE was an encryption algorithm. CASE is used to derive encryption keys, but later it's not involved in packet encryption.